project_task_materials/views/project_view.xml

```diff
@@ -1,25 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-  <data>
+<odoo>
 
     <record id="view_project_task_form_materials" model="ir.ui.view">
-      <field name="name">project.task.form.materials</field>
-      <field name="model">project.task</field>
-      <field name="inherit_id" ref="project.view_task_form2"/>
-      <field name="arch" type="xml">
-
-        <field position="after" name="work_ids">
-          <!--Materials used-->
-          <field name="material_ids" groups="project.group_tasks_work_on_tasks">
-            <tree string="Materials used" editable="top">
-              <field name="product_id"/>
-              <field name="quantity"/>
-            </tree>
-          </field>
+        <field name="name">project.task.form.materials</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Materials used" attrs="{'invisible': [('project_id', '=', False)]}">
```

Comment:
I tried this PR on the OCA Runbot, and the material list was not editable.
I'm not sure if its a problem with Runbot or with the module - can you check?